app/views/housing_rooms/floor_plans.html.erb

```diff
@@ -0,0 +1,7 @@
+<% content_for :header do %>
+  <%= render 'components/page_header', :title => "#{@housing_building.name} FLoor Plans" %>
+<% end %>
+
+<div class="floor-plans"> 
+  <embed src="<%= rails_blob_path(@housing_building.floor_plans) %>" width="1100" height="600" type="application/pdf">
```

Comment:
perhaps we can make this more dynamic to different screen sizes by setting the width to 100%